test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionRangeTests.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
+using Moq;
```

Comment:
This looks like it may be an used using.

Reply:
Removed.